app/src/gui/components/workspace/notebooks.tsx

```diff
@@ -194,17 +198,35 @@ export default function NoteBooks() {
           </Button>{' '}
           tab and click the activate button.
         </Typography>
-        {showCreateNewNotebookButton && (
+        <div
+          style={{display: 'flex', justifyContent: 'space-between', gap: '8px'}}
+        >
+          {showCreateNewNotebookButton ? (
+            <Button
+              variant="contained"
+              onClick={() => history(ROUTES.CREATE_NEW_SURVEY)}
+              sx={{mb: 3, mt: 3, backgroundColor: theme.palette.primary.main}}
+              startIcon={<AddCircleSharpIcon />}
+            >
+              Create New {NOTEBOOK_NAME}
+            </Button>
+          ) : (
+            <div />
+          )}
           <Button
             variant="contained"
-            color="primary"
-            onClick={() => history(ROUTES.CREATE_NEW_SURVEY)}
+            disabled={refresh}
             sx={{mb: 3, mt: 3, backgroundColor: theme.palette.primary.main}}
-            startIcon={<AddCircleSharpIcon />}
+            startIcon={<RefreshOutlined />}
+            onClick={async () => {
```

Comment:
this op is very fast and gives no visible indication to user that it is occurring, can you do a debounced/min duration loading icon or similar? Or even use the global popup provider to flash a little success alert like "{NOTEBOOK_NAME}s refreshed"

Reply:
I like the idea of adding a success alert as it doesn't arbitrarily delay the user refreshing.  Will update